app/_locales/en/messages.json

```diff
@@ -3346,9 +3341,6 @@
   "priorityFee": {
     "message": "Priority fee"
   },
-  "priorityFeeProperCase": {
```

Comment:
Nit: it seems that a bunch of the changes are related to removing this message. It's not clear how this relates to the other changes though. Could this be done in a separate PR? That would help minimize the size of this hotfix.

Reply:
Reverted these changes in latest.

Reply:
or not. thought i got them all. fixing this now

Reply:
Now these changes have been excluded